pyproject.toml

```diff
@@ -28,6 +28,7 @@ pytest = "^7.0.1"
 pytest-cov = "^3.0.0"
 pytest-markdown-docs = "^0.4.3"
 pytest-mock = "^3.10.0"
+requests-mock = "1.10.0"
```

Comment:
Thanks for putting this in the `dev` dependencies.  We want to keep this library quite lightweight.  So be conscious of adding libraries to the main dependencies - especially big ones.